### PR TITLE
[DS-1506] Select image window isn't visible until the user clicks on …

### PR DESCRIPTION
**Pull request by david-storm**
Steps to Reproduce
 
- log in as admin
- create a Landing page LB
- add e.g Grid CTA block
- click on the Add media button
- click on the screen

**Actual Result**
Select image window isn’t visible until the user clicks on the screen.

**Expected Behaviour**
Select image window is displayed right after the user clicks on the Add media button

### CODE REVIEW

**Review by rollki — APPROVED** [review]
In fact, this is a workaround that saves a lot of time, since JS does not correctly calculate the position of the window, so this solution is acceptable in my opinion.

Otherwise, if the community does not agree, then we need to spend some time to understand why JS calculates it incorrectly, but in this case the estimate is not known.

File: assets/scss/layout-builder.scss [hunk 1]
```diff
@@ -557,6 +557,9 @@ figure.drupal-media.ck-widget {
   }
 }
 .ui-dialog {
+  &.entity-browser-modal {
+    top: 8% !important;
```
Comment (froboy) [inline assets/scss/layout-builder.scss:561]: This breaks the ability for a user to reposition the modal on the page in an unexpected way. 

https://github.com/YCloudYUSA/y_lb/assets/238201/2cd28e21-4161-4425-b8d2-11420d285ef4

I'd recommend reviewing https://www.drupal.org/project/issues/entity_browser?text=modal&status=All and finding a more complete solution.
https://github.com/YCloudYUSA/y_lb/assets/238201/2cd28e21-4161-4425-b8d2-11420d285ef4

**Review by froboy — CHANGES_REQUESTED** [review]

File: assets/scss/layout-builder.scss (hunk 1 above)
Comment (podarok) [reply assets/scss/layout-builder.scss:561]: We need less !important statements out there

**Comment by froboy** [comment]
I spent some time trying to resolve this today and it's incredibly challenging. I found a few issues like https://www.drupal.org/project/media_library_edit/issues/3292509 that imply there might be an issue with the core dialog or displace libraries, but I couldn't nail down anything that alleviated the issue. The strange thing is that the dialog is spawned in the wrong place, but when the user clicks anywhere in the page, it's positioned correctly. Even doing something in devtools works to reposition the dialog correctly. :\ Here's a video

https://github.com/user-attachments/assets/66beef15-9d05-42f5-8bad-9ef30c99332b

**Comment by froboy** [comment]
#244 fixed this.

**Pull request closed without merging**